docs/benchmark.md

```diff
@@ -38,12 +38,19 @@ This will
 - Run the benchmark tests using `ginkgo`
 - Output the test results to STDOUT
 
-critest connects to Unix: `unix:///var/run/dockershim.sock` or Windows: `tcp://localhost:3735` by default. For other runtimes, the endpoint can be set by flags `--runtime-endpoint` and `--image-endpoint`.
+critest connects to Unix: `unix:///run/containerd/containerd.sock` or Windows: `tcp://localhost:3735` by default. For other runtimes, the endpoint can be set by flags `--runtime-endpoint` and `--image-endpoint`.
 
 ## Additional options
 
 - `-ginkgo.focus`: Only run the tests that match the regular expression.
 - `-image-endpoint`: Set the endpoint of image service. Same with runtime-endpoint if not specified.
-- `-runtime-endpoint`: Set the endpoint of runtime service. Default to Unix: `unix:///var/run/dockershim.sock` or Windows: `tcp://localhost:3735`.
+- `-runtime-endpoint`: Set the endpoint of runtime service. Default to Unix:
+  `unix:///run/containerd/containerd.sock` or Windows: `tcp://localhost:3735`.
```

Comment:
remove this info (about containerd and docker tcp). Should be something like:

`-runtime-endpoint`: Set the endpoint of runtime service. Default to searching a list of known endpoints.